### ISSUE TITLE
I encountered some tricky problems

### ISSUE THREAD

**Issue by psw13583112** [jfzhang95/PoseAug#22]
Can this model run a single picture? When I ran a picture, there was some misalignment. The model used was MLP trained by PoseAug, and I found that the result was not very satisfactory.
![20210908-104340](https://user-images.githubusercontent.com/40815200/132480759-0c30b5e9-3d16-4ec3-bba7-d74b00c8d6c8.jpg)
![20210908-170632](https://user-images.githubusercontent.com/40815200/132480769-738be91e-62ed-4bb5-93a6-6ce5094fade3.png)
![20210908-170643](https://user-images.githubusercontent.com/40815200/132480772-b033cc62-5b1e-40d6-98e1-f1b785e0caf1.png)

**Comment by Garfield-kh** [comment]
Hi, thank you for the interest!
I think the problem might be 2D pose normalization. The 2D pose here has absolute value > 1 (head, feet), which is not usual in training data.
Can you try scale down the 2D pose by 0.5?

**Comment by psw13583112** [comment]
@Garfield-kh  Thanks a lot for your prompt reply, I tried to reduce it by 0.5 times, the effect is indeed much better, but the results obtained by predicting the same picture every time are not the same.

**Comment by Garfield-kh** [comment]
Did you fix the random seed and also model.eval() for the dropout layer?

**Comment by psw13583112** [comment]
@Garfield-kh Thank you, I will pay attention to these details
